boards/arm/lpcxpresso54114/doc/lpcxpresso54114.rst

```diff
@@ -0,0 +1,161 @@
+.. _lpcxpresso54114:
+
+NXP LPCXPRESSO54114
+#####################
+
+Overview
+********
+
+The LPCXpresso54114 board has been developed by NXP to enable evaluation
+of and prototyping with the LPC54110 family of MCUs and its low power
```

Comment:
change to 

     with the low-power LPC54110 family of MCUs. 

and delete the rest of the sentence starting with "and it's low power..."